.github/workflows/jira.pr.yml

```diff
@@ -0,0 +1,20 @@
+on: 
+  pull_request:
+    branches: 
+    - master
+
+name: Jira Ticket Creation for PRs
```

Comment:
This might be a nit. `Jira` -> `JIRA`.

Reply:
It turns out Atlassian uses "Jira" instead of "JIRA", so it is my PR description that's flawed.

Reply:
Except that I use both JIRA and Jira in the workflow 🤦‍♂